samples/subsys/ipc/openamp_rsc_table/remote/stm32mp157c_dk2.overlay

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2019, STMICROLECTRONICS
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	chosen {
+		/*
+		 * shared memory reserved for the inter-processor communication
+		 */
+		zephyr,ipc_shm = &mcusram3;
+	};
+
+	mcusram3: memory1@10040000 {
+		device_type = "memory";
+		compatible = "mmio-sram";
+		reg = <0x10040000 DT_SIZE_K(64)>;
+	};
+};
+
+&mcusram {
+	reg = <0x10000000 DT_SIZE_K(256)>;
+};
```

Comment:
Missing newline at end of file.

Reply:
@ulfalizer : remarks taken into account, thanks!